packages/react-components/src/components/styles/AttachmentCard.styles.ts

```diff
@@ -41,6 +41,16 @@ export const useAttachmentCardStyles = makeStyles({
   },
   title: {
     width: '100%'
+  },
+  focusState: {
+    '& > * button:focus': {
+      border: `black solid ${_pxToRem(1)}`
```

Comment:
I think FluenUI usually use `outline` for such cases (might be a good idea to check this too). Should we use the outline too?

Reply:
i tried outline as well, but it has lower z index so the border is getting cutoff

Reply:
<img width="411" alt="image" src="https://github.com/Azure/communication-ui-library/assets/109105353/dd1c8515-7c58-4ecf-8087-92bb4415e8aa">